### DISCUSSION TITLE
Screen resolution

### DISCUSSION THREAD

**Discussion by MohamedGamalAzab** [rustdesk/rustdesk#10030] [Q&A]
I created a Python script to capture the screen on windows , and it used to send the resolution as 1366x768.
Now it sends the screen resolution as 1366x557 when close rsutdesk .
How can I fix this issue so it sends the screen resolution as 1366x557 when RustDesk is closed?
I tried changing all the settings related to RustDesk, but the problem still persists

**Comment by fufesou** [comment]
![image](https://github.com/user-attachments/assets/43679e56-04f8-4632-b9b8-3d8e663c802d)

Have you ever changed the resolution here?
